superset-frontend/src/components/Label/index.tsx

```diff
@@ -103,9 +114,12 @@ export default function Label(props: LabelProps) {
     bsStyle: officialBootstrapStyles.includes(props.bsStyle || '')
       ? props.bsStyle
       : 'default',
-    className: officialBootstrapStyles.includes(props.bsStyle || '')
-      ? undefined
-      : `label-${props.bsStyle}`,
+    className: cx({
+      [`${props.className}`]: props.className,
```

Comment:
This bit lets className(s) given to the <Label> be passed through directly.

Reply:
You can also do `cx(props.className, { [label]: condition })`